kinetic/distribution.yaml

```diff
@@ -799,11 +799,11 @@ repositories:
       tags:
         release: release/kinetic/{package}/{version}
       url: https://github.com/yujinrobot-release/ecl_core-release.git
-      version: 0.61.14-0
+      version: 0.61.15-0
     source:
       type: git
       url: https://github.com/stonier/ecl_core.git
-      version: devel
+      version: release/0.61-indigo-kinetic
```

Comment:
Just out of curiosity can you elaborate why you are switching to custom "version-locked" branches? I would like to understand better how / why this is being used.